### PR TITLE
Apply success alert

### PR DESCRIPTION
**Pull request by Werenfeldt**
### Notable changes

Toast when applied is succesa

### Checklist

- [X] I've run the tests and verified that nothing explodes
- [X] I've added a screenshot of the test report
- [X] The code is formatted with a C# formatter
- [X] I reviewed this pr myself

### Test report screenshot

Screenshot 2021-12-21 at 13 47 37
Screenshot 2021-12-21 at 13 47 46

### CODE REVIEW

File: ProjectBank/Client/Pages/ProjectPage.razor [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 @using Microsoft.AspNetCore.Authorization
+@using System.Timers;
 @attribute [Authorize]
 @inject HttpClient Http
 @inject IJSRuntime _jsRuntime
 
 <PageTitle>Project</PageTitle>
 
+@if(show == "show") {
+<div class="alert alert-success alert-dismissible fade show" role="alert" id="SuccessAlert">
```
Comment (JacobMoller) [inline ProjectBank/Client/Pages/ProjectPage.razor:11]: Why did the compiler not indent this? :D

File: ProjectBank/Client/Pages/ProjectPage.razor [hunk 2]
```diff
@@ -50,17 +59,41 @@
...
+                show = "show";
+                StateHasChanged();
+                StartTimerAsync();
+            } 
+        }   
+    }
+
+    public async void StartTimerAsync()
+    {
+      Time = 5;
```
Comment (JacobMoller) [inline ProjectBank/Client/Pages/ProjectPage.razor:86]: What is "Time"? Maybe rename to Seconds or something if that is the unit :D

**Review by JacobMoller — APPROVED** [review]
Approved, but please look at the comments for clean code and naming conventions

File: ProjectBank/Client/Pages/ProjectPage.razor [hunk 3]
```diff
@@ -1,11 +1,20 @@
 ﻿@page "/project/{Id:int}"
 @using Microsoft.AspNetCore.Authorization
+@using System.Timers;
```
Comment (frederikmnielsen) [inline ProjectBank/Client/Pages/ProjectPage.razor:3]:
```suggestion
```

**Review by frederikmnielsen — APPROVED** [review]

**Pull request merged**